src/pages/Error/Error.component.tsx

```
@@ -0,0 +1,17 @@
+import { useNavigate } from 'react-router-dom';
+
+import { ErrorLayout } from '@layouts/ErrorLayout';
+
+export const Error = () => {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)